### PR TITLE
Support hiliting the intersection of models and subcategories (bp #3893)

### PR DESCRIPTION
**Pull request by pmconne**
Backport #3893
(cherry picked from commit c88c97a336c96fa48007b02702c2004f491b9552)

### CODE REVIEW

**Review by bbastings — APPROVED** [review]

**Review by mgooding — APPROVED** [review]

**Review by DStradley — APPROVED** [review]

File: docs/changehistory/NextVersion.md [hunk 1]
```diff
@@ -3,3 +3,19 @@ publish: false
 ---
 # NextVersion
 
+(??)## Hiliting models and subcategories
```
Comment (calebmshafer) [inline docs/changehistory/NextVersion.md:6]: What's up with all the `??`s?

Comment (calebmshafer) [reply docs/changehistory/NextVersion.md:6]: If you're wondering where to put it, can you put these notes in the `3.2.0.md` file? That will just require a new docs deployment to get added in.

Comment (pmconne) [reply docs/changehistory/NextVersion.md:6]: Thanks for catching that. I am using a different diff tool to resolve conflicts on linux, it inserts those.

Comment (pmconne) [reply docs/changehistory/NextVersion.md]: Ok I wondered about NextVersion - will move to 3.2.0.md.

Comment (pmconne) [reply docs/changehistory/NextVersion.md]: Moved.

**Review by calebmshafer — APPROVED** [review]

**Pull request merged**